arches/install/arches-templates/project_name/templates/email/download_ready_email_notification.htm

```diff
@@ -0,0 +1,42 @@
+{% load static %}
+{% load template_tags %}
+{% load i18n %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        {% include 'custom_email_css.htm' %}
+    </head>
+
+    <body>
+
+        {% include 'custom_email_header.htm' %}
+
+        {% if username != "" %}
+            <p>
+                {% blocktrans %}
+                    {{salutation}} {{username}}
```

Comment:
I'm not seeing `{{salutation}}` or `{{expiration}}` defined in any python files, so these values would be blank

Reply:
@chrabyrd At present, yes.  They would need to be defined in the EXTRA_EMAIL_CONTEXT setting which is currently empty in Arches core.

The anticipation is (and it's certainly how we've implemented it on our Arches instance) that these will be set in settings_local.py (this will be explained in documentation).

If you think default settings should be in the core settings.py file, would you be able to give me a steer on what those should be.

Reply:
I've added a default to the project template settings.py with the salutation of "Hi" and then expiration calculated from the CELERY_SEARCH_EXPORT_EXPIRES setting as these are required by the templates and, at the very least, this will give administrators a starting point to set these values.